### PR TITLE
Feature/245

### PR DESCRIPTION
**Pull request by Jahenstein**
- Melvil accepts asset code in two formats e.g. AB123456 or 123456
- Contact me to get proper .env file.

### CODE REVIEW

File: forms/copy.py [hunk 1]
```diff
@@ -6,13 +6,15 @@
 
 from models import Copy
 
-""" Regex return True for assets like "wr123456".
+""" Regex return True for assets like "ab123456" or "123456".
```
Comment (wojwarych) [inline forms/copy.py]: Capitalize those `ab123...` and `123...` so they really match the regex that is used.

File: forms/copy.py [hunk 2]
```diff
@@ -6,13 +6,15 @@
...
 General template:
 two letters (letter case does not matter) + six digits
+or six digits only
 """
-asset_code_regex = Regexp('^[A-Z]{2}[0-9]{6}$',
+asset_code_regex = Regexp('^([A-Z]{2}[0-9]{6}|[0-9]{6})$',
                           flags=re.IGNORECASE,
-                          message='Insert valid asset code.')
+                          message='Insert valid asset code\
+                              eg. ab123456 or 123456')
```
Comment (wojwarych) [inline forms/copy.py]: As earlier

File: forms/copy.py (hunk 1 above)
Comment (Jahenstein) [reply forms/copy.py]: Regexp accepts both capital and non-capital letters, notice ignorecase flag in regexp ;)

Comment (wojwarych) [reply forms/copy.py]: If so, then it's just easier to change the regex itself from the current one to this:
`^([A-Za-z]{2}[0-9]{6}|[0-9]{6})$`
The `[A-Za-z]{2}` part tells that the regex accepts excatly two characters from latin alphabet both uppercase and lowercase. No need to pass a parameter to a function. ;)

**Pull request merged**